.github/workflows/markdown_format.yml

```diff
@@ -0,0 +1,20 @@
+name: markdown formatting
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '**/*'
```

Comment:
issue(blocking): Should start when *.md files are pushed. And I think it should be run on pull request.